resources/js/components/ActiveStatus.js

```diff
@@ -0,0 +1,227 @@
+window.ActiveStatus = class ActiveStatus {
```

Comment:
Did you change something here or is this just the existing code, with other indentation?

And can you please rename the file to `activejourneys`, so its obvious that those two belong together?

Reply:
No, currently there are no changes. I just moved the code to our webpack and put a wrapper class around it.
In the next step I will rewrite it to use the new stopovers table instead of the json.
Sure, I will rename it!